src/styles/main.scss

```diff
@@ -1,14 +1,8 @@
-html {
```

Comment:
u actually need them, because fonts don't look the same with mockup's

Reply:
Done.